scripts/xcode-upload-debug-files.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# print commands before executing them and stop on first error
+set -x -e
+
+echo "warning: uploading debug symbols - set SENTRY_SKIP_DSYM_UPLOAD=true to skip this"
+
+NODE_BINARY=$(command -v node || echo "")
+export NODE_BINARY
+
+IOS_PROJ_PATH=$(pwd)
+
+# Override the default with the global environment
+ENV_PATH="$IOS_PROJ_PATH/.xcode.env"
+if [ -f "$ENV_PATH" ]; then
+    source "$ENV_PATH"
+fi
+
+# Override the global with the local environment
+LOCAL_ENV_PATH="${ENV_PATH}.local"
+if [ -f "$LOCAL_ENV_PATH" ]; then
+    source "$LOCAL_ENV_PATH"
+fi
+
+if [ -z "$NODE_BINARY" ]; then
+  echo "[Warning]: Node path was not found on \`.xcode.env\` and \`.xcode.env.local.\`. " \
```

Comment:
This warning is the only one with this style.

`[Warning]:`

Reply:
Fixed on https://github.com/getsentry/sentry-cordova/pull/372/commits/1ace52258126b71c58bfa551a8248f0a0c7c0d06